### PR TITLE
adding new daylight icon for assignments configured for turnitin

### PR DESCRIPTION
**Pull request by Surekh**

### CODE REVIEW

File: icons.scss [hunk 1]
```diff
@@ -710,7 +710,7 @@
...
+		content: url("data:image/svg+xml,%3Csvg%20width%3D%2218%22%20height%3D%2218%22%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20viewBox%3D%220%200%2018%2018%22%3E%3Cpath%20fill%3D%22%23565a5c%22%20d%3D%22M8%2016v1c0%20.5-.4%201-1%201H6c-.6%200-1-.5-1-1v-1c0-.6.4-1%201-1h1c.6%200%201%20.4%201%201m5%200v1c0%20.5-.4%201-1%201h-1c-.6%200-1-.5-1-1v-1c0-.6.4-1%201-1h1c.6%200%201%20.4%201%201m-5-5v1c0%20.6-.4%201-1%201H6c-.6%200-1-.4-1-1v-1c0-.6.4-1%201-1h1c.6%200%201%20.4%201%201m5%200v1c0%20.6-.4%201-1%201h-1c-.6%200-1-.4-1-1v-1c0-.6.4-1%201-1h1c.6%200%201%20.4%201%201M8%206v1c0%20.6-.4%201-1%201H6c-.6%200-1-.4-1-1V6c0-.6.4-1%201-1h1c.6%200%201%20.4%201%201m5%200v1c0%20.6-.4%201-1%201h-1c-.6%200-1-.4-1-1V6c0-.6.4-1%201-1h1c.6%200%201%20.4%201%201M8%201v1c0%20.6-.4%201-1%201H6c-.6%200-1-.4-1-1V1c0-.5.4-1%201-1h1c.6%200%201%20.5%201%201m5%200v1c0%20.6-.4%201-1%201h-1c-.6%200-1-.4-1-1V1c0-.5.4-1%201-1h1c.6%200%201%20.5%201%201%22%2F%3E%3C%2Fsvg%3E");
```
Comment (Surekh) [inline icons.scss]: I am not sure why this is modified. Happy to exclude it, if that is preferred.

File: icons.scss [hunk 2]
```diff
@@ -2762,7 +2768,7 @@
...
...+xml,%3Csvg%20height%3D%2230%22%20width%3D%2230%22%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20viewBox%3D%220%200%2030%2030%22%3E%0A%20%20%3Cpath%20fill%3D%22%23565a5c%22%20d%3D%22M17.641%2C17.98l-9.89%2C11.5a1.488%2C1.488%2C0%2C0%2C1-1.13.52.277.277%2C0%2C0%2C1-.09-.01.22.22%2C0%2C0%2C1-.08.01%2C1.509%2C1.509%2C0%2C0%2C1-1.01-.39l-4.95-4.5a1.5%2C1.5%2C0%2C1%2C1%2C2.02-2.22l3.9%2C3.55%2C8.95-10.42a1.5%2C1.5%2C0%2C1%2C1%2C2.28%2C1.96Z%22%2F%3E%0A%20%20%3Cpath%20fill%3D%22%23565a5c%22%20d%3D%22M26%2C0H4A4.012%2C4.012%2C0%2C0%2C0%2C0%2C4V20.85a3.5%2C3.5%2C0%2C0%2C1%2C1.505-.35A3.492%2C3.492%2C0%2C0%2C1%2C2%2C20.545V16H7v5H3.3a3.439%2C3.439%2C0%2C0%2C1%2C.556.411L5.6%2C23H6.729L9%2C20.356V16h3.741l1.1-1.282A3.48%2C3.48%2C0%2C0%2C1%2C16%2C13.551V9h5v5H18.3a3.417%2C3.417%2C0%2C0%2C1%2C1.547%2C2H21v5H17.681L14%2C25.28V28H11.661l-1.72%2C2H26a4.012%2C4.012%2C0%2C0%2C0%2C4-4V4A4.012%2C4.012%2C0%2C0%2C0%2C26%2C0ZM7%2C14H2V9H7ZM7%2C7H2V4A2.006%2C2.006%2C0%2C0%2C1%2C4%2C2H7Zm7%2C7H9V9h5Zm0-7H9V2h5Zm7%2C21H16V23h5ZM21%2C7H16V2h5Zm7%2C19a2.006%2C2.006%2C0%2C0%2C1-2%2C2H23V23h5Zm0-5H23V16h5Zm0-7H23V9h5Zm0-7H23V2h3a2.006%2C2.006%2C0%2C0%2C1%2C2%2C2Z%22%2F%3E%3C%2Fsvg%3E%0A%3C%2Fsvg%3E");
```
Comment (Surekh) [inline icons.scss]: Ditto

File: tier1-icons.html [hunk 3]
```diff
@@ -608,9 +608,9 @@
...
-          <path d="M11.7,9.3c-0.4-0.4-1-0.4-1.4,0L4,15.6l-2.3-2.3c-0.4-0.4-1-0.4-1.4,0s-0.4,1,0,1.4l3,3C3.5,17.9,3.7,18,4,18&#xD;&#xA;&#x9;&#x9;s0.5-0.1,0.7-0.3l7-7C12.1,10.3,12.1,9.7,11.7,9.3z"/>
-          <path d="M11,9c0.3,0,0.5,0.1,0.7,0.3c0.4,0.4,0.4,1,0,1.4l-7,7C4.5,17.9,4.3,18,4,18s-0.5-0.1-0.7-0.3l-3-3&#xD;&#xA;&#x9;&#x9;c-0.4-0.4-0.4-1,0-1.4C0.5,13.1,0.7,13,1,13s0.5,0.1,0.7,0.3L4,15.6l6.3-6.3C10.5,9.1,10.7,9,11,9"/>
+          <path d="M17,0H1C0.4,0,0,0.5,0,1v8v2.2C0.3,11.1,0.7,11,1,11s0.7,0.1,1,0.2V10h2v2H3.2L4,12.8l2-2V10h0.8l2.1-2.1&#xA;&#x9;&#x9;C9.2,7.6,9.6,7.3,10,7.2V6h2v1.2c0.4,0.1,0.8,0.4,1.1,0.7C13.7,8.4,14,9.2,14,10h2v2h-2v-2c0,0.8-0.3,1.6-0.9,2.1L11.2,14H12v2h-2&#xA;&#x9;&#x9;v-0.8L7.2,18H9h8c0.6,0,1-0.5,1-1V9V1C18,0.5,17.6,0,17,0z M4,8H2V6h2V8z M4,4H2V2h2V4z M8,8H6V6h2V8z M8,4H6V2h2V4z M12,4h-2V2h2&#xA;&#x9;&#x9;V4z M16,16h-2v-2h2V16z M16,8h-2V6h2V8z M16,4h-2V2h2V4z"/>
```
Comment (Surekh) [inline tier1-icons.html]: ditto.

File: icons.scss (hunk 1 above)
Comment (dlockhart) [reply icons.scss]: Hmm yeah probably best to exclude it

File: tier1-icons.html [hunk 4]
```diff
@@ -212,10 +212,10 @@
...
           <path d="M12.6 0H1.4A1.417 1.417 0 0 0 0 1.43v15.14A1.417 1.417 0 0 0 1.4 18h11.2a1.417 1.417 0 0 0 1.4-1.43v-2.72l-1.39 1.39c-.02.02-1.93 1.76-3.35 1.76a1.22 1.22 0 0 1-.9-.35c-.8-.81-.06-2.08.26-2.63a9.97 9.97 0 0 1 1.16-1.61L14 8.19V1.43A1.417 1.417 0 0 0 12.6 0zM3.03 2H8a1 1 0 0 1 0 2H3.03a1 1 0 0 1 0-2zM7 12.01L3 12a1 1 0 1 1 0-1.99h4a1 1 0 0 1 0 2zM11 8H3a1 1 0 0 1 0-2h8a1 1 0 0 1 0 2z"/>
-		</g>
-		<g id="dragger">
-		  <path d="M8 16v1c0 .5-.4 1-1 1H6c-.6 0-1-.5-1-1v-1c0-.6.4-1 1-1h1c.6 0 1 .4 1 1M13 16v1c0 .5-.4 1-1 1h-1c-.6 0-1-.5-1-1v-1c0-.6.4-1 1-1h1c.6 0 1 .4 1 1M8 11v1c0 .6-.4 1-1 1H6c-.6 0-1-.4-1-1v-1c0-.6.4-1 1-1h1c.6 0 1 .4 1 1M13 11v1c0 .6-.4 1-1 1h-1c-.6 0-1-.4-1-1v-1c0-.6.4-1 1-1h1c.6 0 1 .4 1 1M8 6v1c0 .6-.4 1-1 1H6c-.6 0-1-.4-1-1V6c0-.6.4-1 1-1h1c.6 0 1 .4 1 1M13 6v1c0 .6-.4 1-1 1h-1c-.6 0-1-.4-1-1V6c0-.6.4-1 1-1h1c.6 0 1 .4 1 1M8 1v1c0 .6-.4 1-1 1H6c-.6 0-1-.4-1-1V1c0-.5.4-1 1-1h1c.6 0 1 .5 1 1M13 1v1c0 .6-.4 1-1 1h-1c-.6 0-1-.4-1-1V1c0-.5.4-1 1-1h1c.6 0 1 .5 1 1"/>
-		</g>
+        </g>
```
Comment (Surekh) [inline tier1-icons.html]: Tabs are consistent now, nothing else has changed.

**Pull request merged**